### PR TITLE
fix link to new google amp directory structure

### PR DESCRIPTION
**Pull request by patofp**
@stanch I saw that netlify fails on CI due to a broken link: https://github.com/snowplow/documentation/pull/501 and I saw that the directory structure for the Google AMP has changed: https://github.com/snowplow/documentation/commit/ee5038128c786d625a8836976e1f4215e816de6b#diff-59f254a507df68d58523665fcddad75b41a19d8b53f953b4d23f1d495a1b8f50

This should fix the error.

### CODE REVIEW

**Comment by igneel64** [comment]
Hey @patofp  :) I made this [change](https://github.com/snowplow/documentation/pull/499) 5 days ago, but all the checks were green on the GitHub actions. Is there anything more we need to keep in check?

**Comment by patofp** [comment]
> Hey @patofp :) I made this [change](https://github.com/snowplow/documentation/pull/499) 5 days ago, but all the checks were green on the GitHub actions. Is there anything more we need to keep in check?

Hey @igneel64 , I saw your PR and it is weird because I get this error locally when I run `yarn start` on `main` branch with latest changes, but also on CI on this [PR]( https://github.com/snowplow/documentation/pull/501) could I be missing something?

**Comment by stanch** [comment]
Weird, that build did find the broken link but did not fail for some reason: https://app.netlify.com/sites/snowplow-docs/deploys/64957d52a9603e0008df4430#L89.

@igneel64 you should use `move.sh` to move content. It updates all links and adds a redirect. E.g. `./move.sh docs/old/path docs/new/path`. (You might need to install `moreutils`.)

File: docs/reusable/untitled-reusable-block-38/_index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ There are currently 19 trackers available for setup:
...
+| [AMP Tracker](docs/collecting-data/collecting-from-own-applications/google-amp-tracker/version-1/index.md) | Snowplow is collaborating with Google on their [Accelerated Mobile Pages Project](https://www.ampproject.org/) (AMPP or AMP for short). | ![](https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Maintained&color=9e62dd&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC) |
```
Comment (stanch) [inline docs/reusable/untitled-reusable-block-38/_index.md:6]:
```suggestion
| [AMP Tracker](/docs/collecting-data/collecting-from-own-applications/google-amp-tracker/version-1/index.md) | Snowplow is collaborating with Google on their [Accelerated Mobile Pages Project](https://www.ampproject.org/) (AMPP or AMP for short). | ![](https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Maintained&color=9e62dd&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC) |
```

File: docs/reusable/untitled-reusable-block-38/_index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ There are currently 19 trackers available for setup:
...
+| [AMP Tracker](/docs/collecting-data/collecting-from-own-applications/google-amp-tracker/version-1/index.md) | Snowplow is collaborating with Google on their [Accelerated Mobile Pages Project](https://www.ampproject.org/) (AMPP or AMP for short). | ![](https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Maintained&color=9e62dd&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC) |
```
Comment (stanch) [inline docs/reusable/untitled-reusable-block-38/_index.md:6]:
```suggestion
| [AMP Tracker](/docs/collecting-data/collecting-from-own-applications/google-amp-tracker/index.md) | Snowplow is collaborating with Google on their [Accelerated Mobile Pages Project](https://www.ampproject.org/) (AMPP or AMP for short). | ![](https://img.shields.io/static/v1?style=flat&label=Snowplow&message=Maintained&color=9e62dd&labelColor=9ba0aa&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAeFBMVEVMaXGXANeYANeXANZbAJmXANeUANSQAM+XANeMAMpaAJhZAJeZANiXANaXANaOAM2WANVnAKWXANZ9ALtmAKVaAJmXANZaAJlXAJZdAJxaAJlZAJdbAJlbAJmQAM+UANKZANhhAJ+EAL+BAL9oAKZnAKVjAKF1ALNBd8J1AAAAKHRSTlMAa1hWXyteBTQJIEwRgUh2JjJon21wcBgNfmc+JlOBQjwezWF2l5dXzkW3/wAAAHpJREFUeNokhQOCA1EAxTL85hi7dXv/E5YPCYBq5DeN4pcqV1XbtW/xTVMIMAZE0cBHEaZhBmIQwCFofeprPUHqjmD/+7peztd62dWQRkvrQayXkn01f/gWp2CrxfjY7rcZ5V7DEMDQgmEozFpZqLUYDsNwOqbnMLwPAJEwCopZxKttAAAAAElFTkSuQmCC) |
```

**Review by stanch — APPROVED** [review]

**Pull request merged**